build/android/build.gradle

```diff
@@ -1,10 +1,21 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+project.ext.set("versionMajor", 5) // Version Major
+project.ext.set("versionMinor", 2) // Version Minor
+project.ext.set("versionPatch", 0) // Version Patch
+project.ext.set("versionCode", 24) // Android Version Code, +2 after each release!
```

Comment:
Please state why it must be +2.

Reply:
+1 for arm and +1 for arm64 apks. Each APK must have a larger versioncode than the previous

Reply:
Okay thanks. It would be great if you could put that into the comment so that it's more obvious.

Reply:
Just tell me where to add it.

Reply:
> put that into the comment so that it's more obvious.

Inline. Line 6, comment next to the end of the line. The exact words may be chosen feely.

Reply:
Done.